src/components/new-request/search.vue

```diff
@@ -18,24 +18,27 @@
           @change="clearErrors()"
         >
           <template v-slot:item="{ item }">
-            <v-list-item-title
+            <v-list-item-content
```

Comment:
This "hybrid" menu is a v-select with custom v-list items in it. As this group header is clickable (not like a title), this seems to be a better sub-component for it.

Ref: https://v2.vuetifyjs.com/en/api/v-list-item-content/
(also look at the other v-list-item-xxx items if interested)

Reply:
This fixes the scenario where when we select an action, the select jurisdiction dropdown remains disabled right Sev?

Reply:
Yes, you may have noticed that, sometimes, clicking the menu item did not actually select the item. The padding was the culprit: it caused the v-select to handle the click action instead of the item to handle the action.

Reply:
Ohhh, interesting. OK Sev, makes sense.